tests/apollo/util/skvbc.py

```diff
@@ -433,6 +433,24 @@ async def send_write_kv_set(self, client=None, kv=None, max_set_size=None, long_
         else:
             return writeset[0][0], writeset[0][1]
 
+    async def validate_last_exec_seq_num_for_all_replicas(self, timeout):
+
+        # Validate if all replicas have same last executed sequence number after the write
+        with log.start_action(action_type="validate_last_exec_seq_num_for_all_replicas"):
+            with trio.fail_after(timeout):
+                while True:
+                    repl_counter = 0
+                    last_seq_num0 = await self.bft_network.wait_for_last_executed_seq_num(repl_counter)
+                    while repl_counter < self.bft_network.num_total_replicas() - 1:
+                        repl_counter += 1
+                        last_seq_num1 = await self.bft_network.wait_for_last_executed_seq_num(repl_counter)
+                        if last_seq_num0 != last_seq_num1:
+                            break
```

Comment:
see my next comment, the sleep(0.5) should come here..

Reply:
Fixed